src/packages/src/xroad/ubuntu/generic/xroad-addon-hwtokens.postinst

```diff
@@ -2,4 +2,8 @@
 
 invoke-rc.d --quiet xroad-signer try-restart || true
```

Comment:
Should check state (configure), executing this during abort-upgrade is unnecessary. There might be similar issues in other scripts as well.

Reply:
I think this was addressed in https://github.com/nordic-institute/X-Road/pull/957/commits/4a395e9934c2fca14bcbc5aa115b9251f011a6de